mentoring/tests/integration/base_test.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2014 Harvard
+#
+# Authors:
+#          Xavier Antoviaque <xavier@antoviaque.org>
```

Comment:
Should we extract authors out into a text file, perhaps? There's a lot more than just @antoviaque at this point. Heck, we have git histories that record authorship for us.